client/src/document/index.scss

```diff
@@ -22,13 +22,23 @@
   }
 }
 
+/* override rule from reset as it does not work for MDN */
```

Comment:
Why not? (don't tell me, tell the future by typing in the full explanation into the code comment)